modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -22,6 +22,17 @@ import * as SelectionUtils from '../selection/SelectionUtils';
 import { InsertContentDetails } from './ContentTypes';
 import * as InsertList from './InsertList';
 
+const wrappedElements = [ 'pre' ];
+
+const shouldFlatten = (fragment: AstNode, parentNode: Element) => {
+  const currentNode = fragment.firstChild;
+
+  const isAFlattenableTag = Arr.contains(wrappedElements, currentNode.name);
+  const isPastingInTheSameTag = currentNode.name.toUpperCase() === parentNode.tagName;
```

Comment:
Just for consistency as we normally lowercase the tag name instead in these comparisons.
```suggestion
  const isPastingInTheSameTag = currentNode.name === parentNode.tagName.toLowerCase();
```

In all honesty, we probably should look a making a helper function in say `Strings.ts` for case insensitive comparison. It might be worth logging that as a separate tech debt task?

Reply:
done